### ISSUE TITLE
Mapping a body always reports worth zero credits

### ISSUE THREAD

**Issue by richardbuckle** [EDCD/EDDI#1362]
## EDDI version in which issue is found
3.4.1-rc1

## VoiceAttack version in which issue is found (as applicable)
n/a

## Steps to reproduce
1. Create a brand new speech personality.
1. Edit  `Body report` to `{set reportScanValue to 1}`. 
1. Scan a valuable body.

## Expected
A report containing: `This could be worth [estimate] credits`.

## Observed
Report consistently contains: `This could be worth zero credits`.

## Investigation
eddi_context_last_scan_estimated_value is void in `Report last scan value`.

**Comment by richardbuckle** [comment]
Please see branch `hotfix/1362-bodymapvalues` for a putative fix.

**Comment by Tkael** [comment]
Please see PR #1364 for recommended changes.

**Comment by richardbuckle** [comment]
Sorry, not fixed. I'm still in Podar mapping one body at a time with the current dev build. I'm mapping all the worthwhile bodies in order and so far had done A 5, A 5 a, A 6, A 7, B 5, B 6, B 7. I'm keeping EDSM's dashboard open for comparison.

Upon mapping B8 I got:
Mapping is finished. Efficiency bonus granted.  Planet B 8 is a medium-gravity terraformable High metal content world. I am detecting high levels of active Silicate sub-surface magma flows. Most prevalent in the super dense atmosphere is Carbon dioxide with an average surface temperature of 765 Kelvin. This data could fetch zero credits.  Bodies A 5, A 5 a, A 6, A 7, B 5, B 6, B 7, B 9, B 10, B 11 and B 12 are also recommended candidates to map.
Planet B 8 is a notable High metal content world with potential for terraforming.

Problems: 

1. It used to remember that I had scanned the others in previous sessions, this time it didn't. However that may be a side-effect of my wiping my SQL db. I'll restore it.
2. Still saying zero credits. Should be 572,013 accodig to EDSM.
